### PR TITLE
CEDS-2770 New GTM rollout approach

### PR DESCRIPTION
**Pull request by PauloPortugal**
All frontend HMRC services need to implement a cookie banner and this
changes the way we setup GTM (Google Tag Manager) into our services.
This is a centralised HRMC approach.

The container "a" has been provided by Alex Stanton, our performance
analyst.

### CODE REVIEW

**Review by glidester — APPROVED** [review]

File: test/views/ChoiceViewSpec.scala [hunk 1]
```diff
@@ -67,6 +67,8 @@ class ChoiceViewSpec extends UnitViewSpec with CommonMessages with Stubs with In
                                     |list-of-available-journeys="CRT"
                                     |google-analytics.token=N/A
                                     |google-analytics.host=localhostGoogle
+                                    |gtm.container=a
+                                    |tracking-consent-frontend.url="http://localhost:6796/tracking-consent/tracking.js"
```
Comment (MaciejRewera) [inline test/views/ChoiceViewSpec.scala:71]: For Movements you've used port `12345`. Which one is correct?

File: test/views/declaration/DeclarationChoiceViewSpec.scala [hunk 2]
```diff
@@ -129,6 +129,8 @@ class DeclarationChoiceViewSpec extends UnitViewSpec with CommonMessages with St
                                     |list-of-available-declarations="STANDARD"
                                     |google-analytics.token=N/A
                                     |google-analytics.host=localhostGoogle
+                                    |gtm.container=a
+                                    |tracking-consent-frontend.url="http://localhost:6796/tracking-consent/tracking.js"
```
Comment (MaciejRewera) [inline test/views/declaration/DeclarationChoiceViewSpec.scala:133]: Same here

File: test/views/ChoiceViewSpec.scala (hunk 1 above)
Comment (PauloPortugal) [reply test/views/ChoiceViewSpec.scala]: yes you are correct

Comment (PauloPortugal) [reply test/views/ChoiceViewSpec.scala]: fixed

File: test/views/declaration/DeclarationChoiceViewSpec.scala (hunk 2 above)
Comment (PauloPortugal) [reply test/views/declaration/DeclarationChoiceViewSpec.scala]: fixed

**Review by glidester — APPROVED** [review]

**Pull request merged**